02 Local Platform/01 Key Concepts/01 Getting Started/04 Basic Usage.php

```diff
@@ -20,7 +20,7 @@
     <li>In the Pull QuantConnect Organization Workspace window, create a directory to serve as the workspace and then click <span class="button-name">Select</span>.</li>
     <img class="docs-image" src="https://cdn.quantconnect.com/i/tu/pull-cloud-workspaces.png"></li>
 
-    <?php echo file_get_contents(DOCS_RESOURCES."/cli/init/wsl.php"); ?>
```

Comment:
Please update all the refs to .html and rename the resource since its not doing any dynamic injection.